package.json

```diff
@@ -40,7 +40,6 @@
 		"changelog": "node ./bin/changelog",
 		"changelog:zenhub": "node ./bin/changelog --changelogSrcType='ZENHUB_RELEASE'",
 		"deploy": "npm run build:deploy && sh ./bin/github-deploy.sh",
-		"postdeploy": "npm run storybook:deploy",
```

Comment:
Should this be removed here?

Reply:
Yes, the storybook deployment will be run by `storybook-deploy.sh` https://github.com/woocommerce/woocommerce-blocks/blob/5c7c3e49d94eab0337a35791cf7ff8157944799c/bin/storybook-deploy.sh/#L46-L49